src/story.py

```diff
@@ -6,32 +6,33 @@ class Story(object):
     """
 
     def __init__(self):
-        self.__reset()
-
-
-    def __reset(self):
-        """Clean up the structure"""
-        pass
+        self.items = {}
+        self.traits = {}
+        self.classes = {}
+        self.states = {}
+        self.antonyms = {}
 
 
     def add_item(self, name, in_game_name, description):
         """Add an item to the story"""
-        pass
+        self.items[name] = {'name': in_game_name, 'description': description}
 
 
-    def add_trait(self, name, in_game_name, decription, antonym=None):
-        """Add a trait, aswell as the antonym if it's not None"""
-        pass
+    def add_trait(self, name, in_game_name, description, antonym=None):
+        """Add a trait, as well as the antonym if it's not None"""
+        self.traits[name] = {'name': in_game_name, 'description': description}
+        if antonym is not None:
+            self.antonyms[antonym] = name
 
 
     def add_class(self, class_name, trait_value_dict):
         """Add a class, trait_value_dict is a dictionary 'trait' to 'value'"""
-        pass
+        self.classes[class_name] = {'traits': trait_value_dict}
```

Comment:
the name class might get confusing in the context of code, maybe rename class to role?